tools/block-generator/Makefile

```diff
@@ -1,41 +1,58 @@
-SCENARIO = scenarios/config.allmixed.small.yml
-SKIP = --skip-runner
+SCENARIO = scenarios/benchmarks/stress.50000.yml
+PG_UP = --start-postgres
```

Comment:
This seems to be unused.

Reply:
Good catch!

Reply:
f29b623188d7c07e5929f4d8bd80dbbfcf196a55